### PR TITLE
detect/analyzer: add more details for the tcp window keyword-v2

### PR DESCRIPTION
**Pull request by Nancyenos**
Ticket: 6352

Make sure these boxes are checked accordingly before submitting your Pull Request -- thank you.

## Contribution style:
- [x] I have read the contributing guide lines at
   https://docs.suricata.io/en/latest/devguide/contributing/contribution-process.html

## Our Contribution agreements:
- [x] I have signed the Open Information Security Foundation contribution agreement at
   https://suricata.io/about/contribution-agreement/ (note: this is only required once)

Link to ticket: https://redmine.openinfosecfoundation.org/issues/6352

Describe changes:
-Add more details for the tcp-window keyword
-
-

### Provide values to any of the below to override the defaults.

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/2104

### CODE REVIEW

File: src/detect-engine-analyzer.c [hunk 1]
```diff
@@ -932,6 +933,14 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
...
                 break;
             }
+            case DETECT_WINDOW: {
+                const DetectWindowData *wd = (const DetectWindowData *)smd->ctx;
+                jb_open_object(js, "window");
+                jb_set_uint(js, "size", wd->size);
+                jb_set_bool(js, "negated", wd->negated);
+                jb_close(js);
+                break;
+            }
```
Comment (inashivb) [inline src/detect-engine-analyzer.c:943]: This is looking good. Congrats! :)
I think your suricata-verify PR needs work though

**Review by jufajardini — CHANGES_REQUESTED** [review]
Once the SV work is sorted, you can also rebase the work here and submit a new PR :)

**Comment by Nancyenos** [comment]
> Once the SV work is sorted, you can also rebase the work here and submit a new PR :)

ok.will do

**Comment by Nancyenos** [comment]
work continued in #12035

**Pull request closed without merging**